sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/AnomalyDetectionLiveTests.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Azure.AI.MetricsAdvisor.Tests
 {
+    [Ignore("Data is old and tests can't be recorded again")]
```

Comment:
I know this is too suspicious, but the problem is that I can't record these tests again because the data we're trying to get from the service is not there anymore. My guess is that the service periodically deletes entries that are too old. I'll adjust the date times we use accordingly (and always use a relative time), but I'll do this in a separate PR to make this one easier to review.

Reply:
do you have an issue with that? consider linking the issue to this ignore message so it is easier to map

Reply:
I think I was too sleepy when I did this. The change is just updating some local IDs and rerecording some tests, so I think I can do it in this PR.

Reply:
In the end, the issue was caused because of the configuration we deleted by accident. Talked to the service team and they were able to restore the deleted config, so I removed the `Ignore` attribute.